spec/components/attachment_spec.rb

```diff
@@ -269,19 +269,18 @@ def assert_thumbnail(type, src: nil)
     assert_select '.gem-c-attachment:not([class*="govuk-!-margin-bottom-"])'
   end
 
-  it "adds GA4 tracking on HTML attachment links" do
+  it "adds GA4 tracking on attachment links" do
```

Comment:
Nitpick: maybe rename this test to it `includes GA4 tracking on attachment links by default`